### PR TITLE
contrib/fonts-tex-gyre-otf: new package (2.501)

### PR DESCRIPTION
**Pull request by wezm**
This collection of fonts provides well designed alternatives to some common proprietary fonts/base PostScript fonts such as Helvetica, Palatino, Times, and Century Schoolbook.

~~Had to disable the update check, don't think there's a way to automate it.~~

### CODE REVIEW

**Comment by q66** [comment]
why isn't it possible to automate it? the https://www.gust.org.pl/projects/e-foundry/tex-gyre/whole url contains the latest version string

**Comment by wezm** [comment]
Ahh sorry. Misunderstanding on my part. Will fix tomorrow.

**Comment by wezm** [comment]
Righto, update check fixed.

File: contrib/fonts-tex-gyre-otf/template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+pkgname = "fonts-tex-gyre-otf"
+pkgver = "2.501"
+_pkgver = pkgver.replace(".", "_")
```
Comment (q66) [inline contrib/fonts-tex-gyre-otf/template.py:3]: as there is only one instance of this being used, just put the .replace with original pkgver directly in the source string

File: contrib/fonts-tex-gyre-otf/template.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+pkgrel = 0
+pkgdesc = "TeX Gyre Collection of Fonts"
+maintainer = "Wesley Moore <wes@wezm.net>"
+license = "OFL-1.1"
+url = "https://www.gust.org.pl/projects/e-foundry/tex-gyre/index_html"
+source = f"https://www.gust.org.pl/projects/e-foundry/tex-gyre/whole/tg{_pkgver}otf.zip"
+sha256 = "d7f8be5317bec4e644cf16c5abf876abeeb83c43dbec0ccb4eee4516b73b1bbe"
+
+
+def do_install(self):
```
Comment (q66) [inline contrib/fonts-tex-gyre-otf/template.py:13]: you can shorten this by using install_file with a glob

**Comment by wezm** [comment]
Updates applied.

**Pull request merged**